### PR TITLE
Feature/cid 1879/update readme

### PR DESCRIPTION
**Pull request by mohamedlajmileanix**

### CODE REVIEW

File: src/main/kotlin/net/leanix/vsm/gitlab/broker/webhook/adapter/feign/GitlabWebhookProvider.kt [hunk 1]
```diff
@@ -39,21 +39,26 @@ class GitlabWebhookProvider(
...
-                receiveRepositoryUpdateEvents = true,
-                enableSSLVerification = enableSSLVerification
-            )
-        }.onSuccess {
-            logger.info("Webhook created with id ${it.id}")
-        }.onFailure {
-            logger.error("Error creating webhook: ${it.message}")
-        }.getOrThrow()
+    override fun createWebhook(): GitlabWebhook? {
+        return if (gitlabWebhookUrl.isEmpty() || gitlabWebhookUrl.isNotBlank()) {
```
Comment (geoandri) [inline src/main/kotlin/net/leanix/vsm/gitlab/broker/webhook/adapter/feign/GitlabWebhookProvider.kt:43]: Is this correct? If you get an empty webhook url you cannot create a webhook. I think the `isNotBlank()` should be sufficient.

File: src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/runner/InitialStateRunner.kt [hunk 2]
```diff
@@ -37,12 +39,19 @@ class InitialStateRunner(
     }
 
     private fun setupWebhook() {
-        runCatching {
-            webhookService.registerWebhook()
-        }.onSuccess {
-            logger.info { "webhook registered successfully" }
-        }.onFailure {
-            logger.error(it) { "webhook registration failed" }
+        if (gitlabWebhookUrl.isNotBlank() && gitlabWebhookUrl.isNotEmpty()) {
```
Comment (geoandri) [inline src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/runner/InitialStateRunner.kt:42]: Isn't `isNotBlank()` sufficient? Why do we need to check also for non empty?

Comment (mohamedlajmileanix) [reply src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/runner/InitialStateRunner.kt:42]: you're right, isNotBlank should be enough. I will push a fix.

**Review by geoandri — APPROVED** [review]

File: src/main/kotlin/net/leanix/vsm/gitlab/broker/webhook/adapter/feign/GitlabWebhookProvider.kt (hunk 1 above)
Comment (mohamedlajmileanix) [reply src/main/kotlin/net/leanix/vsm/gitlab/broker/webhook/adapter/feign/GitlabWebhookProvider.kt]: fixed in the latest commit.

File: src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/runner/InitialStateRunner.kt (hunk 2 above)
Comment (mohamedlajmileanix) [reply src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/runner/InitialStateRunner.kt]: fixed in the latest commit.

**Pull request merged**